resized_image_transport/catkin.cmake

```diff
@@ -23,6 +23,8 @@ catkin_package(
 
 if(EXISTS ${jsk_topic_tools_SOURCE_DIR}/cmake/nodelet.cmake)
   include(${jsk_topic_tools_SOURCE_DIR}/cmake/nodelet.cmake)
+elseif(EXISTS ${jsk_topic_tools_SOURCE_PREFIX}/cmake/nodelet.cmake)
+  include(${jsk_topic_tools_SOURCE_PREFIX}/cmake/nodelet.cmake)
 else(EXISTS ${jsk_topic_tools_SOURCE_DIR}/cmake/nodelet.cmake)
   include(${jsk_topic_tools_PREFIX}/share/jsk_topic_tools/cmake/nodelet.cmake)
 endif(EXISTS ${jsk_topic_tools_SOURCE_DIR}/cmake/nodelet.cmake)
```

Comment:
since you have https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_topic_tools/catkin.cmake#L25 line, you can remove ALL include codes, if you have `find_package(catkin COMPONENT jsk_topic_tools)`